docs/installation.md

```diff
@@ -6,6 +6,9 @@ title: Installation
 You can use Scalafmt from your editor, build tool or terminal.
 
 ## IntelliJ
+*For the following paragraphs about IntelliJ setup, please note that the referred (macOS) `Preferences` dialog is named `Settings ` for Windows and Linux.*
```

Comment:
This will be the first thing the user sees after clicking "Getting started" from the landing page

![screenshot 2018-11-16 at 14 18 43](https://user-images.githubusercontent.com/1408093/48623660-92b75280-e9aa-11e8-91af-c445c25ffb80.png)

I prefer to mention it right next to where "Preferences" is mentioned first. How about using "Preferences/Settings > ..." instead?

> `Preferences/Settings > Plugins`

Reply:
This matches the IntelliJ page you linked.

Reply:
Ok then, so I'd stay with you first suggestion, so I'll decorate the first occurrence as: `Preferences > Plugins` (for Windows/Linux it is called `Settings` instead of `Preferences`)

Does that sound good?

Reply:
Sounds good, either way is fine by me 👍

Reply:
here you go :smile: